start_docker_dev.sh

```diff
@@ -1,5 +1,5 @@
 HSUSER=headerservice
-TAGNAME=4.1.2_408-5.1.0-salobj_5.11.0-hs_2.2.0
+TAGNAME=4.1.1-5.1.0-salobj_5.14.0-hs_2.2.0
```

Comment:
Does the HS version here also get updated since you updated other versions or after the release?

Reply:
yes, after it's tagged.